### DISCUSSION TITLE
Can I use events?

### DISCUSSION THREAD

**Discussion by Imlimp** [VonHeikemen/lsp-zero.nvim#119] [Q&A] [answered]
Are there anyway for me to configure this in lsp-zero?
```
local cmp = require('cmp')
cmp.event:on(
  'confirm_done',
  cmp_autopairs.on_confirm_done()
)
```

**Comment by VonHeikemen** [comment, marked as answer]
Yes... kind of. Do exactly that. 

lsp-zero doesn't hide the `cmp` module, you have access to it and you can use it. But do it after lsp-zero's setup.

**Reply by Imlimp** [reply]
Thank you!
